### DISCUSSION TITLE
Internal Server Error when Login via SAML.

### DISCUSSION THREAD

**Discussion by khushal786** [getredash/redash#7193] [Q&A] [answered]
Hi Redash Community, I am a new user of Redash. I set up Redash using AWS AMI. After that, I used a load balancer to redirect my IP to a specific domain.

Also, I have set up SAML SSO for my Azure users by following this [blog](https://medium.com/opstree-technology/azure-ad-integration-with-redash-saml-sso-93b5ef8f3c1d). After doing all the steps in the blog when I am using SAML login it gives me this error 

```
Internal Server Error
The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.
```

Could someone please help me out where I am making mistakes?

**Comment by khushal786** [comment, marked as answer]
Got the solution I have not mentioned the metadata URL and certificate in my env file. So I was facing this error.

**Reply by justinclift** [reply]
Awesome.  That's good info for anyone else that has this problem in the future. :smile:

**Comment by khushal786** [comment]
Thanks ☺️
